aion/web/models/question.ex

```diff
@@ -13,8 +15,17 @@ defmodule Aion.Question do
   Builds a changeset based on the `struct` and `params`.
   """
   def changeset(struct, params \\ %{}) do
-    struct
-    |> cast(params, [:content, :image_name])
-    |> validate_required([:content, :image_name])
+    if params["subject"] do
+      subject = Repo.get(Subject, params["subject"])
+
+      struct
```

Comment:
Maybe we can create separate methods for creating changeset for params that contains and does not contain subject.